example/models/products.go

```diff
@@ -3,17 +3,26 @@ package models
 import "time"
 
 type Product struct {
-	Id         uint64    `json:"id"`
-	Name       string    `json:"name"`
-	MerchantId uint64    `json:"merchant_id"`
-	CategoryId *uint64   `json:"category_id,omitempty"`
-	Tags       []uint64  `json:"tags"`
-	Images     []string  `json:"image_ids"`
-	Sizes      []Sizes   `json:"sizes"`
-	SaleDate   time.Time `json:"sale_date"`
-	EndDate    time.Time `json:"end_date"`
+	Id         uint64     `json:"id"`
+	Name       string     `json:"name"`
+	MerchantId uint64     `json:"merchant_id"`
+	CategoryId *uint64    `json:"category_id,omitempty"`
+	Tags       []uint64   `json:"tags"`
+	Images     []string   `json:"image_ids"`
+	Sizes      []Sizes    `json:"sizes"`
+	SaleDate   time.Time  `json:"sale_date"`
+	EndDate    *time.Time `json:"end_date"`
+
+	RecursiveTest *Product          `json:"recursive_test"`
+	SwagnoTagTest int               `json:"swagno_tag_test" swagno:"object"`
+	TestMap       map[string]*Sizes `json:"test_map"`
+	InterfaceTest interface{}       `json:"interface_test"`
```

Comment:
Are all these test names supposed to be for future tests? May be more clear for users who will reference these example in the future if we moved these over to our testing section instead of being in our examples section.

Reply:
Of course, we can move them in tests later. I was testing on UI with them to see the outcome easily